lib/logstash/inputs/jdbc.rb

```diff
@@ -4,26 +4,55 @@
 require "logstash/plugin_mixins/jdbc"
 require "yaml" # persistence
 
-# This plugin was created as a way to iteratively ingest data in any database
+# This plugin was created as a way to ingest data in any database
 # with a JDBC interface into Logstash. You can periodically schedule ingestion
 # using a cron syntax (see `schedule` setting) or run the query one time to load
 # data into Logstash. Each row in the resultset becomes a single event.
 # Columns in the resultset are converted into fields in the event.
 #
-# ==== Sample usage:
+# ==== Drivers
+#
+# This plugin does not come packaged with JDBC driver libraries. The desired 
+# jdbc driver library must be explicitly passed in to the plugin using the
+# `jdbc_driver_library` configuration option.
+# 
+# ==== Scheduling
+#
+# Input from this plugin can be scheduled to run periodically according to a specific 
+# schedule. This scheduling syntax is powered by [rufus-scheduler](https://github.com/jmettraux/rufus-scheduler)
+#
+# Further documentation describing this syntax can be found [here](https://github.com/jmettraux/rufus-scheduler#parsing-cronlines-and-time-strings)
+#
```

Comment:
Can we add one example here `* * * 5 *` and then people can follow the link if they want to know more.